### PR TITLE
Nopol actor

### PR DESCRIPTION
**Pull request by danglotb**
NopolActor is responsible to run Nopol remotely for the ui-intellij.

### CODE REVIEW

File: nopol/src/main/java/fr/inria/lille/repair/actor/NoPolActor.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    static ActorSystem system;
+    static ActorRef actorNopol;
+
+    public static void main(String[] args) {
+        try {
+            Main.initJSAP();
+        } catch (JSAPException e) {
+            e.printStackTrace();
```
Comment (danglotb) [inline nopol/src/main/java/fr/inria/lille/repair/actor/NoPolActor.java]: remove + throws Exception in method

File: nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+class InternalNopolActor extends UntypedActor {
+
+    @Override
+    public void onReceive(Object message) throws Exception {
+        if (message instanceof String) {
+            String[] args = ((String) message).split(" ");
+            System.out.println(Arrays.toString(args));
+            List<Patch> patches = Collections.EMPTY_LIST;
+            if (Main.parseArguments(args)) {
+                File[] sourceFiles = new File[Config.INSTANCE.getProjectSourcePath().length];
```
Comment (danglotb) [inline nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java]: singleton incompatible with multiple Nopol. first refactor Config.

File: nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.List;
+
+/**
+ * Created by bdanglot on 9/16/16.
+ */
+class InternalNopolActor extends UntypedActor {
+
+    @Override
+    public void onReceive(Object message) throws Exception {
+        if (message instanceof String) {
```
Comment (danglotb) [inline nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java]: use config object instead of String

**Comment by danglotb** [comment]
add a test

File: nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java (hunk 2 above)
Comment (danglotb) [reply nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java]: see #11

File: nopol/src/main/java/fr/inria/lille/repair/actor/NoPolActor.java (hunk 1 above)
Comment (danglotb) [reply nopol/src/main/java/fr/inria/lille/repair/actor/NoPolActor.java]: Done.

File: nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java (hunk 3 above)
Comment (danglotb) [reply nopol/src/main/java/fr/inria/lille/repair/actor/InternalNopolActor.java]: Done.

**Review by tdurieux — APPROVED** [review]

**Pull request merged**